Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -430,6 +447,7 @@ extension PostCampaignCheckoutViewController: STPApplePayContextDelegate {
     completion: @escaping StripeApplePay.STPIntentClientSecretCompletionBlock
   ) {
     guard let paymentIntentClientSecret = self.applePayPaymentIntent else {
+      self.viewModel.inputs.checkoutTerminated()
```

Comment:
These calls aren't needed - AFAIK everywhere that `completion(nil, error)` is called will end up below in `case: .error` of `func applePayContext(didCompleteWith:error:)`. Tested this with a breakpoint and by breaking one of the calls.

Reply:
Oh, thanks! That makes this nicer :) I wasn't sure how to generate one of these errors so I really appreciate that!

Reply:
Ha, me neight, but I just set something to `nil` and checked what happened 😆